src/test/java/seedu/address/logic/commands/QueryDoctorCommandTest.java

```diff
@@ -0,0 +1,113 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.DON;
+import static seedu.address.testutil.TypicalPersons.JAMES;
```

Comment:
For the  TypicalPersons, it'd be nice if we kept the alphabetical naming conventions... so after Don, we should come up with someone who's name starts with E, instead of James...